### PR TITLE
Added Avatar Component

### PR DESCRIPTION
**Pull request by chirag0527**

### CODE REVIEW

File: Components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    width: 2rem;
+    border-radius: 999px;
+    position: relative;
+}
+
+.avatar-xs{
+    height: 1rem;
+    width: 1rem;
+    border-radius: 999px;
+    position: relative;
```
Comment (srishti-maurya) [inline Components/Avatar/avatar.css:34]: Suggestion: there is a repeat of properties for `border-radius: 999px;` and `position: relative;`, maybe you can add other class specific to it

**Review by srishti-maurya — APPROVED** [review]

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
+.avatar-std{
+    height: 5rem;
+    width: 5rem;
+    border-radius: 999px;
+    position: relative;
```
Comment (manojasarsa) [inline Components/Avatar/avatar.css:5]: You can make a separate class for "border-radius" and "position", as it been reused.

File: Components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+.avatar-image-xs{
+    height: 1rem;
+    width: 1rem;
+    border-radius: 999px;
+}
+
+
+.status-dnd{
+    background-color: rgb(255,0,0);
```
Comment (manojasarsa) [inline Components/Avatar/avatar.css:69]: Color Variable can be used.

**Review by manojasarsa — APPROVED** [review]
Rest looks good! 👍

**Comment by nshanbhag20001412** [comment]
hey, can you add a live preview for this? It would be better and easier to review

File: Components/Avatar/avatar.css (hunk 1 above)
Comment (chirag0527) [reply Components/Avatar/avatar.css]: Done! I couldn't come up with a class name so I clubbed together all the classes which were using those 2 specific lines of code.

File: Components/Avatar/avatar.css (hunk 3 above)
Comment (chirag0527) [reply Components/Avatar/avatar.css:62]: I made these files separately, but I have implemented variables in the final downloadable CSS file.

**Comment by chirag0527** [comment]
> hey, can you add a live preview for this? It would be better and easier to review

Sure. I couldn't add the images in codepen but here you go
https://codepen.io/chirag0527/pen/VwrbMmm

**Pull request merged**